packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -143,12 +143,25 @@ class Directionality extends InheritedWidget {
 /// This is more efficient than adding and removing the child widget from the
 /// tree on demand.
 ///
-/// ## Opacity Animation
+/// ## Opacity animation
 ///
 /// Animating an [Opacity] widget directly causes the widget (and possibly its
 /// subtree) to rebuild each frame, which is not very efficient. Consider using
 /// an [AnimatedOpacity] instead.
 ///
+/// ## Transparent image
+///
+/// If only a single [Image] needs to be composited with an opacity between 0.0
+/// and 1.0, it's much faster to directly use [Image] as following:
+///
+/// ```dart
+/// Image(
+///   image: image,
+///   color: Color.fromRGBO(255, 255, 255, opacity),
+///   colorBlendMode: BlendMode.modulate
+/// )
+/// ```
```

Comment:
btw, sample code should always be in a sample section so that the analyzer catches it. looks like this sample code ended up not in a sample code section.

Reply:
@Hixie : what's my option here? Since it's already in `## Transparent image` section, can I create a sub-section `### Sample code`?